config/settings.yml

```diff
@@ -21,3 +21,4 @@ feature_flags:
   cycle_ending_soon: false
   cycle_has_ended: false
   display_apply_button: true
+  improved_locations: false
```

Comment:
Is this name adequate? Should it be called something else?

Reply:
Perhaps `ucas_only_locations`?

Reply:
And maybe that flag should encompass what’s covered in these cards?

* https://trello.com/c/NxrO8GoV
* https://trello.com/c/ErVvUSIA

So, with flag set to `false`:
* as is now (map and codes shown on course detail page).

When set to `true`:
* no map or location codes table shown on course detail page
* selecting UCAS apply route takes you to UCAS apply page (https://trello.com/c/ErVvUSIA)
* the UCAS apply page shows the location codes table (https://trello.com/c/NxrO8GoV)

Essentially, these 3 changes constitute one feature.

Reply:
I've noticed that the card for this PR doesn't mention hiding the location codes table or changing the position of the apply button (as per the design history entry). I don't see that carded up elsewhere, and it makes sense to include those changes here.

Will reopen this PR once those are accounted for. cc: @davidgisbey

Reply:
I think we really need to wait for the ttw changes before we can remove the locations from the page too.